### PR TITLE
Yessenia Rodriguez js data model

### PR DESCRIPTION
**Pull request by yesseniarod**

### CODE REVIEW

File: javascript-data-model/order-history.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    orderNumber: '113-2883177-2648248',
+    deliveryDate: 'Delivered Jul 5, 2020',
+    message: '',
+    title: 'The Art of Sql',
+    author: 'Faroult, Stephane',
+    return: 'Return window closed on Aug 4, 2020',
+    cost: '$33.99',
+    orderDetails: '',
+    invoice: ''
+  }
```
Comment (codingcodymiller) [inline javascript-data-model/order-history.js:76]: For this portion, there is no way for me to know that these two items were in the same order. The main object being stored in the `orderHistory` array should be describing the order itself, and then you can store details about each item ordered in a property on that object

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work so far! Please see my note:

File: javascript-data-model/order-history.js (hunk 1 above)
Comment (yesseniarod) [reply javascript-data-model/order-history.js]: Screen Shot 2021-01-26 at 12 36 06 PM

File: javascript-data-model/order-history.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
+var orderHistory = [
+  {
+    orderPlaced: 'August 4, 2020',
+    items: ['book'],
```
Comment (codingcodymiller) [inline javascript-data-model/order-history.js:4]: This should be holding objects, with each object describing each individual item. For instance:
```
items: [
 {
   type: 'book',
   name: "The Art of Sql"
  }
]
```

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Yessenia! Please revise your format to match the format I described in my note

**Review by codingcodymiller — APPROVED** [review]
Great work!! 👏

**Pull request merged**